pkg/queryfrontend/roundtrip.go

```diff
@@ -22,13 +21,15 @@ const (
 	labelQueryRange = "query_range"
 )
 
+// NewTripperware returns a Tripperware configured with middlewares to
+// limit, align, split, retry and cache requests.
+// Not using the cortex one as it uses  query parallelisations based on
+// storage sharding configuration and query ASTs.
```

Comment:
Perfect! Would you mind putting `cache requests` before `retry`? Because cache middleware is before retry. But just a nit.

Reply:
done